bundles/automation/org.eclipse.smarthome.automation.module.core/META-INF/MANIFEST.MF

```diff
@@ -5,6 +5,7 @@ Bundle-SymbolicName: org.eclipse.smarthome.automation.module.core
 Bundle-Version: 0.9.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Import-Package: com.google.common.collect,
+ org.apache.commons.lang,
```

Comment:
I missed the mf file, sorry, I've seen that the usage of StringUtils was remove from code.